src/clients/memberclient.cpp

```diff
@@ -267,6 +286,28 @@ int main(int argc, char** argv)
   auto ack =
     app.add_subcommand("ack", "Acknowledge self added into the network");
 
+  NodeInfo node_info;
```

Comment:
Could we use a single `node.json` file to pass all the node info in one go here instead? A bit like https://github.com/microsoft/CCF/blob/master/src/genesisgen/main.cpp#L143-L146